src/Umbraco.Cms.Api.Management/ViewModels/Webhook/CreateWebhookRequestModel.cs

```diff
@@ -0,0 +1,6 @@
+namespace Umbraco.Cms.Api.Management.ViewModels.Webhook;
+
+public class CreateWebhookRequestModel : WebhookModelBase
+{
+    public Guid? Id { get; set; }
```

Comment:
@Zeegaan does `UpdateWebhookRequestModel` also need this and `WebhookResponseModel`?

Should it be part of `WebhookModelBase` instead then?

Reply:
No just the CreateModel! 😁
It's basically just so you can specify an Id upon creation 👍

Reply:
Okay, but we should probably add it to `WebhookResponseModel` model as well then - just like with datatype:
https://github.com/umbraco/Umbraco-CMS/blob/268babf8895e0088591ebb4e4efa15112749baa3/src/Umbraco.Cms.Api.Management/ViewModels/DataType/DataTypeResponseModel.cs#L5

Reply:
I added `Guid` to `WebhookResponseModel`:
https://github.com/umbraco/Umbraco-CMS/pull/15147/commits/500513870b6986e147fb89a8e17a944bf33f2d7c

I noticed other response models inherits from `NamedItemResponseModelBase` which inherits from `ItemResponseModelBase` and has `Guid Id`.

E.g. `MemberGroupItemResponseModel` inherits from `NamedItemResponseModelBase`.

Reply:
I guess `WebhookResponseModel` can inherits from `NamedItemResponseModelBase` as webhook inherits `TEntity` or from `ItemResponseModelBase` ?

Reply:
`ItemResponseModelBase` are for the item endpoints, so no more inheritance please 🙈